gradle/wrapper/gradle-wrapper.properties

```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.1.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.1-rc-1-all.zip
```

Comment:
Is this to align the version used internally at Dropbox?

Reply:
nope, figured since this is an open source project I'd use it to test new tooling. Let me know if this gets annoying :-)

Reply:
No but `6.1.1` is the latest 😄. Or did you mean the use `6.2-rc-1`?

Reply:
oh that's weird, it was a studio auto correct, will change